src/Chapter1.hs

```diff
@@ -520,8 +522,9 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
-
+closestToZero x y = if abs (x) == abs(y) then error "values are the same, please choose separate values!" 
+  else if abs (x) > abs (y) then y else x
+-- How could I implemnt another
```

Comment:
If values are the same, you can return either of them. The task doesn't specify which one is better so you can simplify the implementation 👍🏻

Reply:
Okay, changed it 😄